app/styles/front.scss

```diff
@@ -708,33 +686,60 @@ legend {
   width: 1px;
 }
 
-input:checked + .radioButton, input:checked + .radioButton:active:hover {
+.inputButton {
```

Comment:
Not so dirty to me. Having a difference in structure imply a styling difference sounds kind of normal. It's true that there is some ambiguity here, but it's not blocking to me.